spec/support/disability_compensation_form/saved_claim_with_4142.json

```diff
@@ -3,4 +3,4 @@
":[{\"confinementBeginDate\":\"2018-03-29\",\"confinementEndDate\":\"2018-03-29\",\"verifiedIndicator\":false}]},\"disabilities\":[{\"name\":\"Diabetes mellitus\",\"classificationCode\":\"string\",\"disabilityActionType\":\"INCREASE\",\"ratedDisabilityId\":\"0\",\"diagnosticCode\":5235,\"secondaryDisabilities\":[{\"name\":\"Diabetes mellitus\",\"classificationCode\":\"\",\"disabilityActionType\":\"NONE\",\"specialIssues\":[{\"code\":\"TRM\",\"name\":\"Personal Trauma PTSD\"}],\"ratedDisabilityId\":\"\",\"diagnosticCode\":5235}]}],\"treatments\":[{\"center\":{\"name\":\"string\",\"type\":\"DOD_MTF\",\"country\":\"USA\",\"city\":\"string\",\"state\":\"OR\"},\"startDate\":\"2018-03-29\",\"endDate\":\"2018-03-29\"}],\"specialCircumstances\":[{\"name\":\"string\",\"code\":\"string\",\"needed\":false}],\"standardClaim\":false,\"form4142\":{\"privacyAgreementAccepted\":true,\"limitedConsent\":\"true\",\"providerFacility\":[{\"providerFacilityName\":\"provider 1\",\"treatmentDateRange\":[{\"from\":\"1980-1-1\",\"to\":\"1985-1-1\"},{\"from\":\"1986-1-1\",\"to\":\"1987-1-1\"}],\"providerFacilityAddress\":{\"street\":\"123 Main Street\",\"street2\":\"1B\",\"city\":\"Baltimore\",\"state\":\"MD\",\"country\":\"USA\",\"postalCode\":\"21200-1111\"}},{\"providerFacilityName\":\"provider 2\",\"treatmentDateRange\":[{\"from\":\"1980-2-1\",\"to\":\"1985-2-1\"},{\"from\":\"1986-2-1\",\"to\":\"1987-2-1\"}],\"providerFacilityAddress\":{\"street\":\"456 Main Street\",\"street2\":\"1B\",\"city\":\"Baltimore\",\"state\":\"MD\",\"country\":\"USA\",\"postalCode\":\"21200-1111\"}},{\"providerFacilityName\":\"provider 3\",\"treatmentDateRange\":[{\"from\":\"1980-3-1\",\"to\":\"1985-3-1\"},{\"from\":\"1986-3-1\",\"to\":\"1987-3-1\"}],\"providerFacilityAddress\":{\"street\":\"789 Main Street\",\"street2\":\"1B\",\"city\":\"Baltimore\",\"state\":\"MD\",\"country\":\"USA\",\"postalCode\":\"21200-1111\"}},{\"providerFacilityName\":\"provider 4\",\"treatmentDateRange\":[{\"from\":\"1980-4-1\",\"to\":\"1985-4-1\"},{\"from\":\"1986-4-1\",\"to\":\"1987-4-1\"}],\"providerFacilityAddress\":{\"street\":\"101 Main Street\",\"street2\":\"1B\",\"city\":\"Baltimore\",\"state\":\"MD\",\"country\":\"USA\",\"postalCode\":\"21200-1111\"}},{\"providerFacilityName\":\"provider 5\",\"treatmentDateRange\":[{\"from\":\"1980-5-1\",\"to\":\"1985-5-1\"},{\"from\":\"1986-5-1\",\"to\":\"1987-5-1\"}],\"providerFacilityAddress\":{\"street\":\"102 Main Street\",\"street2\":\"1B\",\"city\":\"Baltimore\",\"state\":\"MD\",\"country\":\"USA\",\"postalCode\":\"21200-1111\"}}]},\"overflowText\":\"VA Form 21-4142/4142a has been completed by the applicant and sent to the PMR contractor for processing in accordance with M21-1 III.iii.1.D.2.\",\"claimantCertification\":true,\"autoCestPDFGenerationDisabled\":false,\"applicationExpirationDate\":\"2015-08-28T19:53:45+00:00\",\"directDeposit\":{\"accountType\":\"CHECKING\",\"accountNumber\":\"9876543211234\",\"routingNumber\":\"042102115\",\"bankName\":\"Comerica\"}}}" }
```

Comment:
Ditto the question about why this is flat

Reply:
This one is out of the scope of our modules we are responsible for so I'm more hesitant to modify this one especially since it appears to be string inside of json etc.